lms/urls.py

```diff
@@ -1012,6 +1013,15 @@
         include('lms.djangoapps.certificates.apis.urls', namespace='certificates_api')),
 ]
 
+# Fetch assessments score api
+urlpatterns += [
+    url(r'^api/get_assessments_score/{}/(?P<chapter_id>[^/]*)/'.format(
```

Comment:
If possible, we should remove get from url, HTTP GET request is enough to indicate that

Reply:
Removed
@zahid-arbisoft